_posts/2022-09-03-aphp-user-story.adoc

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: 'https://github.com/quarkusio/quarkusio.github.io.git.'
+date: 2021-09-13
+tags: user-story
+synopsis: 'Quarkus adoption by APHP (Assistance Publique des Hôpitaux de Paris).'
```

Comment:
It would be nice to improve this description, like, what the reader would expect while reading this user store